{{cookiecutter.project_slug}}/.env.example

```diff
@@ -1,6 +1,6 @@
 DEBUG=1
 HOST=localhost
-PORT=8000
+DJANGO_PORT=8000
```

Comment:
This can't be renamed, it's an env var set automatically by Heroku.

Reply:
It can now, since we use a separate `Dockerfile.heroku` for Heroku and that will reference `$PORT` whilst `Dockerfile` will reference `$DJANGO_PORT` so we don't have to do voodoo with node